src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/ciagkpi/PefCiagKpiService.kt

```diff
@@ -31,39 +37,76 @@ class PefCiagKpiService(
   private val telemetryService: TelemetryService,
   private val timelineService: TimelineService,
   private val timelineEventFactory: TimelineEventFactory,
+  private val reviewService: ReviewService,
 
 ) : CiagKpiService() {
 
+  /**
+   * Process an prisoner admission.
+   *
+   * - If the prisoner is brand new create a new Induction schedule
+   * - If the prisoner already has an incomplete induction schedule (this will most likely be
+   * a duplicate message) then do nothing.
+   * - If the prisoner already has had their Induction created. Then this person is likely to
+   * be a coming back into prison and needs to resume their Reviews.
+   */
   override fun processPrisonerAdmission(prisonNumber: String, prisonAdmittedTo: String, eventDate: Instant) {
     log.info { "Creating or updating induction schedule for prisoner [$prisonNumber]" }
 
-    // Check if an induction schedule already exists.
-    val existingSchedule = inductionSchedulePersistenceAdapter.getInductionSchedule(prisonNumber)
-    if (existingSchedule != null) {
-      log.info { "Induction schedule already exists for prisoner [$prisonNumber], ignoring this message." }
-      return
-    }
+    if (activeInductionScheduleAlreadyExists(prisonNumber)) return
```

Comment:
This was previously if any induction schedule exists then ignore the message. Don't think that was correct there will always be an induction schedule but we only want to return when there is an active induction schedule. 

Although even that may not be correct, I'm kind of assuming this will only happen when there is a duplicate admission message and we would want to drop the message in that case but... what if the induction was not complete before the prisoner was released... Could feasibly happen. Then on readmission months or years later there will still be that active induction schedule lying around with an old deadline date.